tests/browser/tsconfig.json

```diff
@@ -1,18 +1,25 @@
 {
-    "extends": "../../src/tsconfig",
+    "$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
Is there any documentation on this key?

Reply:
https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.8.1.1
https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#schema